olive/passes/onnx/optimum_merging.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from typing import Any, Dict, Union
+
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import CompositeOnnxModel, ONNXModel
+from olive.passes import Pass
+from olive.passes.onnx.common import get_external_data_config
+from olive.passes.pass_config import PassConfigParam
+from pathlib import Path
+
+from optimum.onnx import merge_decoders
+import onnx
```

Comment:
better not import the optimum at top level since it would make it to be requirement of Olive instead of an extra